src/main/java/com/client/config/ScheduledEventConfig.java

```diff
@@ -0,0 +1,119 @@
+import org.springframework.scheduling.quartz.MethodInvokingJobDetailFactoryBean;
+import org.springframework.scheduling.quartz.SchedulerFactoryBean;
+
+import java.text.ParseException;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+// TODO: Maybe make scheduled tasks in multiple different folders
+@Configuration
+public class ScheduledEventConfig {
+
+    private final BullhornData bullhornData;
+
+    private final ApplicationSettings appSettings;
+
+    private final EventWorkflowFactory eventWorkflowFactory;
+
+    private final CustomSubscriptionSettings customSubscriptionSettings;
+
+    ScheduledEventConfig(BullhornData bullhornData,
+                         ApplicationSettings appSettings,
+                         EventWorkflowFactory eventWorkflowFactory,
+                         CustomSubscriptionSettings customSubscriptionSettings) {
+        this.bullhornData = bullhornData;
+        this.appSettings = appSettings;
+        this.eventWorkflowFactory = eventWorkflowFactory;
+        this.customSubscriptionSettings = customSubscriptionSettings;
+    }
+
+    @Bean
+    SchedulerFactoryBean mainScheduler(CronTrigger standardCronTriggerBean) {
+        SchedulerFactoryBean mainScheduler = new SchedulerFactoryBean();
+        mainScheduler.setOverwriteExistingJobs(true);
+        mainScheduler.setAutoStartup(true);
+
+        CronTrigger[] customCronTriggers = getCustomCronTriggers();
+        CronTrigger[] cronTriggers = Arrays.copyOf(customCronTriggers, customCronTriggers.length + 1);
+        cronTriggers[customCronTriggers.length] = standardCronTriggerBean;
+        mainScheduler.setTriggers(cronTriggers);
+
+        return mainScheduler;
+    }
+
+    @Bean
+    public CronTrigger standardCronTriggerBean(
+            MethodInvokingJobDetailFactoryBean standardFactoryBean,
+            @Value("${standardCronExpression}") String cronExpression) throws ParseException {
+
+        CronTriggerFactoryBean cronTriggerFactoryBean = new CronTriggerFactoryBean();
+        cronTriggerFactoryBean.setCronExpression(cronExpression);
+        cronTriggerFactoryBean.setJobDetail(standardFactoryBean.getObject());
+        cronTriggerFactoryBean.afterPropertiesSet();
+
+        return cronTriggerFactoryBean.getObject();
+    }
+
+    @Bean
+    public MethodInvokingJobDetailFactoryBean standardFactoryBean() {
+        MethodInvokingJobDetailFactoryBean factoryBean = new MethodInvokingJobDetailFactoryBean();
+
+        factoryBean.setTargetBeanName("standardSubscriptionScheduledEventProcessing");
+        factoryBean.setTargetMethod("run");
+        factoryBean.setConcurrent(true);
+
+        return factoryBean;
+    }
+
+    @Bean
+    ScheduledEventProcessing standardSubscriptionScheduledEventProcessing(@Value("${standardSubscriptionName}") String subName) {
+        return new ScheduledEventProcessing(subName, bullhornData, appSettings, eventWorkflowFactory);
+    }
```

Comment:
The code for the standard subscription is just duplicated in the `getCustomCronTriggers()`. I'm just going to integrate these together